apps/remix-ide/src/app/tabs/runTab/contractDropdown.js

```diff
@@ -29,12 +30,14 @@ class ContractDropdownUI {
       var contractNames = document.querySelector(`.${css.contractNames.classNames[0]}`)
       contractNames.innerHTML = ''
       if (success) {
-        this.selectContractNames.removeAttribute('disabled')
+        this.enableContractNames(true)
+        this.atAddress.removeAttribute('disabled')
```

Comment:
can you use `enableAtAddress` here (and below)?

Reply:
yes, I refactor that later and fergot, thanks